handlers/clusters.go

```diff
@@ -442,6 +440,26 @@ func CreateClusterResponse(params clusters.CreateClusterParams) middleware.Respo
 	// TODO ties into cluster status, make a note if the service is missing
 	sc.Create(&websv.Service)
 
+	// Wait for the replication controllers to exist before building the response.
+	rcc := client.ReplicationControllers(namespace)
+	{
+		var mrepl, wrepl *kapi.ReplicationController
+		mrepl = nil
+		wrepl = nil
+		for i := 0; i < 4; i++ {
+			if mrepl == nil {
+				mrepl, _ = getReplController(rcc, clustername, masterType)
+			}
+			if wrepl == nil {
+				wrepl, _ = getReplController(rcc, clustername, workerType)
+			}
+			if wrepl != nil && mrepl != nil {
+				break
+			}
+			time.Sleep(250 * time.Millisecond)
+		}
```

Comment:
should we have an error clause here in case either `mrepl` or `wrepl` is `nil` ?